### DISCUSSION TITLE
Output is missing + Good Example of How to Ask Questions

### DISCUSSION THREAD

**Discussion by Akki-Rawat** [Jay-study-nildana/FrontEndForStudents#25] [Q&A]
Hey Jay sir

I was trying to practice the react list-component code as you teach us in the class, my code is correct but my output is missing, and I don't know why😶?

Can you please help me out in this? I know there are lots of ways to show the output, I just want to know that if my code is correct they why I'm not able to see the output properly?

**Here is the link of my repo**
[https://github.com/Akki-Rawat/React-Error](https://github.com/Akki-Rawat/React-Error)

_note_:- I also describe it properly in the readme file so check it also if you got confused what to do!

Thanks 
Akki

**Comment by Jay-study-nildana** [comment]
@Akki-Rawat 

it says, repo does not exist. Perhaps, it is private? you have to invite me first. Or just make it public ;) 

(Also, any chance you can put your project on code sandbox? That way, I can check the code directly. If possible, not mandatory)

Thanks

Jay

**Comment by Jay-study-nildana** [comment]
![AkshatReactError1](https://user-images.githubusercontent.com/14297315/236770273-13cf1a32-dad8-4453-aa1f-d506ca3cf7cc.jpg)

here is what I get when I try to open it.

**Comment by Akki-Rawat** [comment]
> ![AkshatReactError1](https://user-images.githubusercontent.com/14297315/236770273-13cf1a32-dad8-4453-aa1f-d506ca3cf7cc.jpg)
> 
> here is what I get when I try to open it.

Sir I send you the invite for the repo.

**Comment by Jay-study-nildana** [comment]
@Akki-Rawat 

Okay, now, I am able to see the code. 

also, look at this discussion on "how to ask react doubts"

https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/24

**Comment by Jay-study-nildana** [comment]
@Akki-Rawat 

Now, this is a very good README.MD. 

Very nice, this is how you should ask questions, I like it. Very good. 

Only thing is, learn to start asking doubts through code sandbox also.

**Comment by Jay-study-nildana** [comment]
@Akki-Rawat 

I have updated your code so it should work now (IN THE FUTURE, PLEASE SHARE YOUR ERRORS ON CODE SANDBOX, and only use GITHUB for super serious private projects which cannot be put on codesandbox)

So, the main issue is this. You are doing like this.

```
<Item key ={item.id} item={item} />
```

but, if you want to show some item, you should ACTUALLY show it. like this

```
                            <ListGroup.Item key={item.id}> 
                                {item.id}. {item.name} and power is : {item.power}
                            </ListGroup.Item>
```

And, you have to practice proper "indentation" my friend. half your problems get solved if you are properly indenting code. 

So, let me know if you have more doubts, create a new issue or new discussion. This issue is resolved and I am closing it. 

Jay

**Comment by Jay-study-nildana** [comment]
Also, an article for students to read. 

https://medium.com/projectwt/the-simple-yet-tough-to-nail-art-of-asking-coding-doubts-d2bffbf4abf5

Thanks

Jay (Vijayasimha BR)
Coding Tutor. Career and Life Coach
----
Discord: Jay_HisManyHobbies#6392
